README.md

```diff
@@ -112,31 +120,28 @@ defmodule MyApp.Web.FallbackController do
 end
 ```
 
-See the section "Overriding `action/2` for custom arguments" in [the Phoenix.Controller docs](https://hexdocs.pm/phoenix/Phoenix.Controller.html) for a clean way to pass in the `user` to each action.
-
 ## Composable Actions
 
 The concept of an authorized action is encapsulated by the `Bodyguard.Action` struct. It can be initialized with defaults, modified during the request cycle, and finally executed in a controller or socket action.
 
 This example is exactly equivalent to the above:
 
 ```elixir
+# lib/my_app/web/controllers/post_controller.ex
 defmodule MyApp.Web.PostController do
   use MyApp.Web, :controller
   import Bodyguard.Action       # Import act/1, permit/3, run/2, etc.
 
   action_fallback MyApp.Web.FallbackController
   
   def index(conn, _) do
-    user = # get current user
-
-    act(MyApp.Blog)             # Initialize a %Bodyguard.Action{}
-    |> put_user(user)           # Assign the user
-    |> permit(:list_posts)      # Check with MyApp.Blog.authorize/3 callback
-    |> run(fn action ->         # Job only executed if authorization passes
+    act(MyApp.Blog)                        # Initialize a %Bodyguard.Action{}
```

Comment:
As noted in the readme, the example is exceptionally verbose for demonstration. Right now, `Bodyguard.Action` is pretty useless, I agree, but I have some future plans.

Think about `conn` and how that's currently the atomic unit of a request in Phoenix. Then realize that a lot of things about `conn` don't jive well when calling directly into contexts from a controller: maps with string parameters, important assigns like `current_user` hidden away – things like that. Wouldn't it be cool if you could just go `Plug.Conn` -> `Bodyguard.Action` -> `context` and Do The Thing(tm), having all the authentication and authorization performed, the appropriate resources fetched, and types coerced and validated?